doc/releases.rst

```diff
@@ -386,6 +385,6 @@ Luminous, only every other stable release was an "LTS" release.
 Therefore,
 
 * Upgrades are supported from Jewel -> Kraken -> Luminous and Jewel -> Luminous.
-* Upgrades beyond Luminous must stop at Luminous.
-* Jewel will be maintained until Mimic is released in the Spring of 2018.
+* Upgrades beyond Luminous must stop at Luminous (e.g., Kraken -> Luminous -> Mimic but not Kraken -> Mimic).
```

Comment:
must first go through an upgrade of luminous before proceeding further... (or something in these lines)